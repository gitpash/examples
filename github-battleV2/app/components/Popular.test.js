import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";
import Popular, { RepoGrid, SelectLanguage } from "./Popular";

describe("Popular", () => {
  it("renders", () => {
    const div = document.createElement("div");
    ReactDOM.render(<Popular />, div);
  });
  test("snapshots", () => {
    const component = renderer.create(<Popular />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});

// describe("SelectLanguage", () => {
//   const props = {
//     lang: ["All", "Javascript", "Ruby", "Java", "CSS", "Python"]
//   };
//   it("renders", () => {
//     const div = document.createElement("div");
//     ReactDOM.render(<SelectLanguage props />, div);
//   });
//   test("snapshots", () => {
//     const component = renderer.create(<SelectLanguage props />);
//     let tree = component.toJSON();
//     expect(tree).toMatchSnapshot();
//   });
// });
