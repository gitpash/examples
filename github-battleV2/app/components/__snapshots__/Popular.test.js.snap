// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Popular snapshots 1`] = `
<div>
  <ul
    className="languages"
  >
    <li
      onClick={[Function]}
      style={
        Object {
          "color": "#d0021b",
        }
      }
    >
      All
    </li>
    <li
      onClick={[Function]}
      style={null}
    >
      Javascript
    </li>
    <li
      onClick={[Function]}
      style={null}
    >
      Ruby
    </li>
    <li
      onClick={[Function]}
      style={null}
    >
      Java
    </li>
    <li
      onClick={[Function]}
      style={null}
    >
      CSS
    </li>
    <li
      onClick={[Function]}
      style={null}
    >
      Python
    </li>
  </ul>
  <p
    style={
      Object {
        "fontSize": "35px",
        "textAlign": "center",
      }
    }
  >
    Loading
  </p>
</div>
`;
